client/app/_components/ui/footer/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from 'next/link'
+import { RiCalendarFill } from 'react-icons/ri'
+import { RiHomeFill } from 'react-icons/ri'
+import { RiMoneyEuroCircleFill } from 'react-icons/ri'
+
+// TODO: This design may have some room for improvement
+const Footer = () => {
```

Comment:
I personally prefer to put Footer/Header/SettingDropdown/Logo under `@_components/layout` instead of `@_components/ui`!

Reply:
True! I fixed it!
https://github.com/HAK2024/fairy-share/pull/55/commits/3a529d251a47866b04fa849ee9f98bd37ca0431e